### PR TITLE
05 controllers specs

### PR DESCRIPTION
**Pull request by gak92**
In this pull request:

- [X] Create request spec for all controllers
- [X] Check if the response is correct
- [X] Check if the template renders correctly
- [X] Check if the response includes placeholder text

### CODE REVIEW

File: spec/requests/posts_spec.rb [hunk 1]
```diff
@@ -2,9 +2,34 @@
...
+      get '/users/1/posts/1'
+    end
+    it 'Response will be successful.' do
+      expect(response).to have_http_status(:ok)
+    end
+    it 'Template render correctly' do
+      expect(response).to render_template(:show)
+    end
+    it 'Response should include right placeholder text.' do
+      expect(response.body).to include('Post details will be shown here')
```
Comment (prolajumokeoni) [inline spec/requests/posts_spec.rb]: - [ ] Kindly make sure that tests written for index and show actions for User and Post controllers are passing, the tests are currently failing see screenshot below ⬇️  


Screenshot_511

File: spec/requests/users_spec.rb [hunk 2]
```diff
@@ -2,9 +2,34 @@
...
+      get '/users/1'
+    end
+    it 'Response will be successful.' do
+      expect(response).to have_http_status(:ok)
+    end
+    it 'Template render correctly' do
+      expect(response).to render_template(:show)
+    end
+    it 'Response should include right placeholder text.' do
+      expect(response.body).to include('User details will be shown here')
```
Comment (prolajumokeoni) [inline spec/requests/users_spec.rb]: - [ ] Kindly make sure that tests written for index and show actions for User and Post controllers are passing, the tests are currently failing see screenshot above ⬆️

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @gak92 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used 
✔️  Good PR title and Description



## Required Changes ♻️

- [ ] Kindly update your readme to have the instructions for running tests locally 👍🏿 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by gak92** [comment]
@prolajumokeoni,
I don't understand why it is failing on your end.
Here is the screenshot of all my test are passing:
image

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**